libkbfs/journal_server_util.go

```diff
@@ -139,6 +143,15 @@ func fillInJournalStatusUnflushedPaths(ctx context.Context, config Config,
 		for ub := range unflushedBytes {
 			jStatus.UnflushedBytes += ub
 		}
+		// Pick the latest end estimate.
```

Comment:
> latest

Just to make sure I understand correctly, there would be exactly 1 item in `endEstimates` when we reach here, right?

Reply:
That's correct.